system/bootstrap.php

```diff
@@ -11,6 +11,13 @@
  * the LICENSE file that was distributed with this source code.
  */
 
+/**
+ * ---------------------------------------------------------------
+ *
+ * @deprecated 4.5.0 This file is no longer used. Moved to Boot.php.
+ * ---------------------------------------------------------------
+ */
+
```

Comment:
What about a helpful upgrade error message in here in case this file is still called?

Reply:
Added.